web/frontend/package.json

```
@@ -13,9 +13,9 @@
     "build-npm": "vite build --config vite.lib.config.ts && npm run build-types",
     "check": "svelte-check --tsconfig ./tsconfig.json",
     "preview": "vite preview",
-    "_prettier": "prettier --ignore-path=.eslintignore \"**/*.{ts,js,cjs,svelte,json,html,md}\"",
+    "_prettier": "prettier \"**/*.{ts,js,cjs,svelte,json,html,md}\"",
     "_eslint": "eslint \"**/*.{ts,js,svelte}\"",
-    "lint": "npm run _eslint",
+    "lint": "npm run _prettier --check && npm run _eslint",
     "format": "npm run _prettier --write && npm run _eslint --fix",
```

```suggestion
    "format": "npm run _prettier -- --write && npm run _eslint --fix",
```

I think we need the extra `--` to pass args through a `npm run` script.

---

Thanks for catching!